src/aiida/cmdline/commands/cmd_computer.py

```diff
@@ -730,3 +730,62 @@ def computer_config_show(computer, user, defaults, as_option_string):
             else:
                 table.append((f'* {name}', '-'))
         echo_tabulate(table, tablefmt='plain')
+
+
+@verdi_computer.group('export', cls=LazyConfigureGroup)
+def computer_export():
+    """Exports the Authinfo details for a computer (and user)."""
+
+
+@computer_export.command('setup')
+@arguments.COMPUTER()
+@arguments.OUTPUT_FILE(type=click.Path(exists=False))
```

Comment:
And, more general, some idea for the future: We could also make the `OUTPUT_FILE` optional, and use `f{label}-setup/config.yaml` as default (also for the code export), as this is probably how people would usually name the output file?

Reply:
Makes sense, but are you suggesting that it is created in a folder? Or should it just create a file?